src/modules/ethereum/exchangeTransactions.js

```diff
@@ -78,3 +78,16 @@ export async function cancelOrderTx(exchange, buy, orderId) {
 
     return { txName, transactionHash };
 }
+
+export async function getSimpleBuyCalc(token, isBuy, rate) {
+    const exchange = await store.getState().web3Connect.augmint.exchange;
+    const orderBook = await exchange.getOrderBook();
```

Comment:
The `orderBook` should be taken from the state (should be there already since the order book panel uses it), rather than loading it every time.